Document/0x05a-Platform-Overview.md

```diff
@@ -542,7 +542,7 @@ An Android certificate must have a validity period that's longer than that of th
 The goal of the signing process is to associate the app file (.apk) with the developer's public key.  To achieve this, the developer calculates a hash of the APK file and encrypts it with their own private key. Third parties can then verify the app's authenticity (e.g., the fact that the app really comes from the user who claims to be the originator) by decrypting the encrypted hash with the author’s public key and verifying that it matches the actual hash of the APK file.
 
 Many Integrated Development Environments (IDE) integrate the app signing process to make it easier for the user. Be aware that some IDEs store private keys in clear text in configuration files; double-check this in case others are able to access such files and remove the information if necessary.
-Apps can be signed from the command line with the 'apksigner' tool provided by the Android SDK (API 24 and higher) or the Java JDK tool 'jarsigner' (for earlier Android versions). Details about the whole process can be found in official Android documentation; however, an example is given below to illustrate the point.
+Apps can be signed from the command line with the 'apksigner' tool provided by the Android SDK (API 24 and higher). It is located at `[SDK-Path]/build-tools/[version]`. Before the apksigner, you can use the Java JDK tool 'jarsigner' for earlier Android versions. Details about the whole process can be found in official Android documentation; however, an example is given below to illustrate the point.

```

Comment:
Change 

> Before the apksigner, you can use the Java JDK tool 'jarsigner' for earlier Android versions.

to something like.

> For API 24.0.2 and below, you can use 'jarsigner', which is part of the Java JDK tool.


And change `(API 24 and higher)` to `(API 24.0.3 and higher)`  (https://developer.android.com/studio/command-line/apksigner)